### PR TITLE
60 navigation tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

Refactored navigation tests to run with angular 10. Separated hamburger menu tests in own test file

medic/[angular10-migration #60](https://github.com/medic/angular10-migration/issues/60)

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by newtewt** [comment]
@ngaruko  can you change the base this is going to? IT should not be `master` should be `4750-angular-10-take-2`

File: tests/conf.js [hunk 1]
```diff
@@ -133,11 +132,11 @@ const login = browser => {
   browser.driver.findElement(by.id('login')).click();
   // Login takes some time, so wait until it's done.
   const bootstrappedCheck = () =>
-    element(by.css('.app-root.bootstrapped')).isPresent();
+    element(by.css('.inbox.bootstrapped')).isPresent();
```
Comment (newtewt) [inline tests/conf.js:135]: The `element(by.css('.app-root.bootstrapped')).isPresent();` works here. I wonder if your change was before the fix was merged. Can you update?

File: tests/conf.js [hunk 2]
```diff
@@ -133,11 +132,11 @@ const login = browser => {
...
   return browser.driver.wait(
     bootstrappedCheck,
-    20 * 1000,
-    'Login should be complete within 20 seconds'
+    40 * 1000,
```
Comment (newtewt) [inline tests/conf.js:138]: I think you were having trouble with your machine. Is this still necessary?

File: tests/e2e/navigation/hamburger-menu.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  });
+
+  it('should open Report bug', () => {
+    commonElements.openMenu();
+    commonElements.checkReportBug();
+  });
+
+  it('should open Configuration app', () => {
+    commonElements.goToConfiguration();
+    const display = element(by.css('[ui-sref="display.date-time"]'));
```
Comment (newtewt) [inline tests/e2e/navigation/hamburger-menu.js:38]: Please move the element definitions to page object.

**Review by newtewt — CHANGES_REQUESTED** [review]
Solid move to separate the tests. A few items to update.

File: tests/e2e/navigation/hamburger-menu.js (hunk 3 above)
Comment (ngaruko) [reply tests/e2e/navigation/hamburger-menu.js]: @newtewt - Restore the selectors and timeouts that were failing due to my environment. Removed hamburger menu (`hamburger-menu.js`) from this PR. Will work on them separately.

**Review by craig-landry — APPROVED** [review]
@ngaruko I've verified the updates from @newtewt's comments. This just needs that conf.js conflict resolved and it's good to go. Please resolve that and let me know and I'm good to reapprove.

**Comment by ngaruko** [comment]
> @ngaruko I've verified the updates from @newtewt's comments. This just needs that conf.js conflict resolved and it's good to go. Please resolve that and let me know and I'm good to reapprove.

Conflict resolved - Merged on local on local and pushed

**Pull request merged**